MouseGuard/mouseguard.html

```diff
@@ -33,6 +33,9 @@
                         <td><label for="attr_cloak_color">Cloak Color:</label></td>
                         <td><input type="text" name="attr_cloak_color" /></td>
                     </tr>
+					<tr>
+						<td><label for"attr_specialty">Specialty:</label/</td>
+						<td><input type="text" name="attr_specialty" /></td>
```

Comment:
You're not closing your <tr> here. This should be fine, since it is the last row in the table, but it should be closed nonetheless.